test/parallel/test-dns-resolve-promises.js

```diff
@@ -4,13 +4,11 @@ require('../common');
 const assert = require('assert');
 const { internalBinding } = require('internal/test/binding');
 const cares = internalBinding('cares_wrap');
-const EPERM = internalBinding('constants').os.errno.EPERM;
+const EPERM = internalBinding('uv').UV_EPERM;
```

Comment:
```suggestion
const { UV_EPERM } = internalBinding('uv');
```

or
```suggestion
const { UV_EPERM: EPERM } = internalBinding('uv');
```
?

Reply:
If you pick one, please use the former, as `UV_EPERM === -EPERM` generally ;)

Reply:
Sure, done!